pkg/detectors/dockerhub/dockerhub.go

```diff
@@ -54,14 +57,23 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 
 	for _, resUserMatch := range userMatches {
 		for _, resAccessTokenMatch := range accessTokenMatches {
+			if len(resAccessTokenMatch) != 2 {
+                        	continue
+                	}
+                	pat := strings.TrimSpace(resAccessTokenMatch[1])
```

Comment:
Why is trimming necessary here? I don't see any way to capture whitespace in the access token pattern. Am I misreading it?